apps/core-app/src/renderer/src/modules/auth/useAuth.ts

```
@@ -635,13 +653,23 @@ async function retryPendingBrowserLogin(): Promise<void> {
     pendingBrowserLogin.value.openedAt
   )
   try {
+    authLoadingState.loginStage = 'preparing'
     await openBrowserLoginPage()
+    authLoadingState.loginStage = 'waiting'
     toast.info('已重新打开登录页面')
   } catch {
```

ESLint

[error] Closing curly brace appears on the same line as the subsequent block.

(style/brace-style)